website/templates/project/project.mako

```diff
@@ -29,7 +29,7 @@
                     <div class="btn-group">
                     % if not node["is_public"]:
                         <button class='btn btn-default disabled'>Private</button>
-                        % if 'admin' in user['permissions']:
+                        % if 'admin' in user['permissions'] and not node['pending_embargo']:
```

Comment:
While this is fine, there should also be a check on the backend that prevents nodes that are pending embargo from being made public.

@HarryRybacki Can you make this change in a separate feature branch?